solvcon/mesh.pyx

```diff
@@ -221,6 +221,17 @@ cdef class Mesh:
             if vals[0] != vals[1]:
                 msgs.append('body(%d,%d)' % vals)
             if msgs:
+                f = open("/tmp/solvcon.debug.log", "a")
```

Comment:
For reading more easily.
It's sometimes hard for me to read the print-out parallelly returned by every worker. The print-out of stdout is a bit randomly. I dumped some of them to a file so I could focus on it easily.

In the next PR to add the debug code, this action won't be applied.